easybuild/easyblocks/t/tensorflow.py

```diff
@@ -112,25 +113,47 @@ def configure_step(self):
                 filtered_path = os.pathsep.join([p for fil in path_filter for p in path if fil not in p])
                 env.setvar(var, filtered_path)
 
+        use_mpi = self.toolchain.options.get('usempi', False)
+
         # put wrapper for Intel C compiler in place (required to make sure license server is found)
         # cfr. https://github.com/bazelbuild/bazel/issues/663
         if self.toolchain.comp_family() == toolchain.INTELCOMP:
-            wrapper_dir = os.path.join(tmpdir, 'bin')
+            icc_wrapper_dir = os.path.join(tmpdir, 'bin')
 
             icc_wrapper_txt = INTEL_COMPILER_WRAPPER % {
                 'compiler_path': which('icc'),
+                'intel_mpi_root': '',
```

Comment:
@akesandgren I would use a more meaningful value here, like `UNKNOWN` or `IRRELEVANT`, in case someone inspects the generated wrapper?